apps/server/src/modules/sharing/uc/share-token.uc.spec.ts

```diff
@@ -948,6 +989,77 @@ describe('ShareTokenUC', () => {
 			});
 		});
 
+		describe('when parent is a columnboard', () => {
+			const setup = () => {
+				const course = courseFactory.buildWithId();
+				courseService.findById.mockResolvedValueOnce(course);
+
+				const columnBoard = columnBoardFactory.build();
+				const user = userFactory.buildWithId();
+				authorization.getUserWithPermissions.mockResolvedValueOnce(user);
+
+				const payload: ShareTokenPayload = { parentType: ShareTokenParentType.ColumnBoard, parentId: columnBoard.id };
+				const shareToken = shareTokenFactory.build({ payload });
+				service.lookupToken.mockResolvedValueOnce(shareToken);
+
+				return { user, shareToken, course, columnBoard };
+			};
+			it('should get token from service', async () => {
```

Comment:
Very small: "should call ShareTokenService to get the token". It doesn't test if it really gets a token